### PR TITLE
Add Visual Studio and Atom syntax highlighters

### PR DESCRIPTION
**Pull request by timothyhinrichs**
Both these can be generated from the Textmate version.  Worth committing?

### CODE REVIEW

File: misc/syntax/atom/language-rego/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## 0.1.0 - First Release
```
Comment (tsandall) [inline misc/syntax/atom/language-rego/CHANGELOG.md]: I will remove this after merging unless you think it's worth keeping.

File: misc/syntax/visualstudio/rego/language-configuration.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (tsandall) [inline misc/syntax/visualstudio/rego/language-configuration.json]: I will replace this with https://github.com/tsandall/vscode-rego/blob/master/language-configuration.json after merging unless you tell me not to.

Comment (tsandall) [inline misc/syntax/visualstudio/rego/language-configuration.json]: The only difference AFAICT is the line comments and quotes.

File: misc/syntax/visualstudio/rego/vsc-extension-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Welcome to your VS Code Extension
```
Comment (tsandall) [inline misc/syntax/visualstudio/rego/vsc-extension-quickstart.md]: Will remove this as well unless you think it's worth keeping.

**Comment by tim-styra** [comment]
All sounds good to me

**Pull request merged**